bases/rest-api/src/clojure/realworld/rest_api/main.clj

```diff
@@ -1,8 +1,32 @@
 (ns clojure.realworld.rest-api.main
   (:require [clojure.realworld.rest-api.api :as api]
+            [clojure.realworld.log.interface :as log]
```

Comment:
I try to have alphabetical order in the imports. It could be nice to change this:
```suggestion
  (:require [clojure.realworld.log.interface :as log]
            [clojure.realworld.rest-api.api :as api]
```

Reply:
Oh, missed the suggestion. Made the change though, so all good I think. 😄